web-client/src/presenter/computeds/formattedTrialSessionDetails.js

```diff
@@ -45,6 +52,10 @@ export const formattedTrialSessionDetails = (get, applicationContext) => {
       const hasNoActiveCases =
         isEmpty(allCases) || isEqual(allCases, inactiveCases);
 
+      if (formattedTrialSession.formattedEligibleCases.length === 0) {
```

Comment:
🤔 I'm wondering if it would be easier to tell when/if the filter was disabled if this check and the check on line 30 were combined?

Reply:
Thanks...done